src/main.jsx

```diff
@@ -30,6 +30,10 @@ const router = createBrowserRouter([
     path: "/cart",
     element: <Cart/>
   },
+  {
+    path: "/home",
```

Comment:
Okay, so I haven't worked with router a lot (really just this project lol), but I see that on line 13-15 we have a path to App, and I know we have used that in other places in the code base. Can we have two paths to the same page? Will both take us home?

Reply:
Ah I read the router details too quickly and skipped over the `<App />` route, just saw the `<ErrorPage />` under the `/` path.

I just tested it out and using either the `/` or '/home' paths work, I'd prefer to get rid of the `/home` path though just to cut back on unneeded redundancy. I'll close this and submit a new PR shortly here.

First time using router here too so learning as we go haha.